docs/runtime-contract.md

```diff
@@ -82,9 +82,9 @@ serverless platform SHOULD observe the following properties:
 
```

Comment:
I cannot comment on it here, but lines 73-74 state:

"containers which are operated as part of a serverless platform SHOULD observe the following properties:"

The use of the keyword here should be replaced with a non keyword.

Reply:
yep good catch - fixed